crates/teloxide-macros/src/fields_parse.rs

```diff
@@ -139,12 +139,12 @@ fn parser_with_separator<'a>(
                 let res = #res;
 
                 match splitted.next() {
-                    Some(d) => ::std::result::Result::Err(teloxide::utils::command::ParseError::TooManyArguments {
+                    Some(d) if !s.is_empty() => ::std::result::Result::Err(teloxide::utils::command::ParseError::TooManyArguments {
                         expected: #expected,
                         found: #expected + 1,
```

Comment:
This `+ 1` is probably wrong, `splitted` might be longer than that 🤔

Reply:
We can use .count() on splitted, but idk what to do with message field then. Is changing the message a non-breaking change?

Reply:
I think so